src/sentry/static/sentry/app/components/buttons/button.jsx

```diff
@@ -0,0 +1,107 @@
+import React, {PropTypes} from 'react';
+import {Link} from 'react-router';
+import classNames from 'classnames';
+
+import '../../../less/components/button.less';
+
+const Button = React.createClass({
```

Comment:
Yeah that's definitely a downside to this, it's more error prone when using, but it seems a bit simpler since it's just a key name vs key + value.

```javascript
<Button priority="primary" size="small">button</Button>
<Button primary small>button</Button>
```
@ckj and I leaned towards boolean props, but isn't a big deal to me, @ckj any strong opinions?